apps/graphql/src/apps/itinerary/types/enums/SortSearch.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import { GraphQLEnumType } from 'graphql';
+
+export default new GraphQLEnumType({
```

Comment:
Values should be a lowercase, in that case you don't need `...(input.sort && { sort: input.sort.toLowerCase() }),` https://github.com/kiwicom/margarita/pull/577/files#diff-cc82f3aaf9876008c3725e8182cfc1f3R64

check the `apps/graphql/src/apps/location/types/enums/LocationType.js` for example

Reply:
Done!